scene/gui/tab_bar.h

```diff
@@ -210,6 +211,9 @@ class TabBar : public Control {
 	void set_clip_tabs(bool p_clip_tabs);
 	bool get_clip_tabs() const;
 
+	void set_tab_style_v_flip(bool p_tab_style_v_flip);
+	bool get_tab_style_v_flip() const;
```

Comment:
This seems unused. If something is not a property, it doesn't need a corresponding getter.